docs/Salesforce-batchsource.md

```diff
@@ -18,10 +18,10 @@ Configuration
 
 **Password:** Salesforce password.
 
-**Client Id:** Application Client Id. This is also called "Consumer Key" in Salesforce UI.
-To create Client Id user needs to create a connected application in Salesforce first.
+**Consumer Key:** Application Consumer Key. This is also known as "Client Id".
+To create Consumer Key user needs to create a connected application in Salesforce first.
 
-**Client Secret:** Application Client Secret. This is also called "Consumer Secret" in Salesforce UI.
+**Consumer Secret:** Application Client Secret. This is also known as "Client Secret".
 To create Client Secret user needs to create a connected application in Salesforce first.
```

Comment:
A Salesforce connected application must be created in order to get a consumer secret.

Reply:
done